src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/multiples/NotificationsExcelReportService.java

```diff
@@ -121,6 +123,8 @@ private static void setCaseDataWithExtractedDocument(MultipleData multipleData,
                 .documentBinaryUrl(documentSelfPath + BINARY)
                 .build();
 
+        log.warn("Doc url - {}", uploadedDocumentType.getDocumentBinaryUrl());
```

Comment:
I assume this is accidental and for local logging?

Reply:
Oops you are correct. Will remove